packages/react-components/react-infobutton/src/stories/Infobutton/InfoButtonCustomIcon.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { Link } from '@fluentui/react-components';
+import { bundleIcon, QuestionCircle12Regular, QuestionCircle12Filled } from '@fluentui/react-icons';
+
+export const CustomIcon = () => {
+  const Icon = bundleIcon(QuestionCircle12Filled, QuestionCircle12Regular);
+
+  return (
+    <InfoButton
+      trigger={{ icon: <Icon /> }}
+      content={
+        <>
+          Popover above-start lorem ipsum dolor sit amet consectetur.{' '}
+          <Link href="https://react.fluentui.dev">Learn more</Link>
+        </>
+      }
+    />
+  );
+};
+
+CustomIcon.parameters = {
+  docs: {
+    description: {
+      story: `An InfoButton can be customized with a custom icon.`,
```

Comment:
In the design figma this example has a disclaimer "use sparingly", I am open to remove this example, but at the same time I would prefer to give the guidance for `bundleIcon`.

Reply:
Is there something specific about `InfoButton` and `bundleIcon` that you want to demonstrate? If not, perhaps demoing `bundleIcon` in the `Icon` docs is better?

Reply:
Maybe just link to the `bundleIcon` example in the doc site?

Reply:
I was thinking about that, the only thing is that we actually don't encourage users to change it. So I think it's better to remove it and I'll add the best practice in a future PR.